project_task_materials_analytic_partner/__openerp__.py

```diff
@@ -19,6 +19,4 @@
         'views/project_view.xml',
     ],
     'installable': True,
-    'autoinstall': True,
```

Comment:
autoinstall is desirable to assure that the functionality is present if you are already working with "other partner" concept. As the default value for the other partner is to be empty, the default behavior is not changed when autoinstalling this module.